rascaline/Cargo.toml

```diff
@@ -58,6 +58,7 @@ approx = "0.5"
 
 [dev-dependencies]
 criterion = "0.5"
+half = "2.2.1"
```

Comment:
maybe merge this with the block below about `clap`?
```suggestion
half = "=2.2.1"
```

Reply:
Yes, that was the plan I don't why this is gone.